src/js/server/login.js

```diff
@@ -50,8 +42,8 @@ fluid.defaults("gpii.express.user.login.post.handler", {
     method: "post",
     invokers: {
         handleRequest: {
-            func: "{reader}.get",
-            args: ["{that}.options.request.body"]
+            funcName: "gpii.express.user.login.post.handler.verifyPassword",
+            args: ["{that}", "{gpii.express.user.utils}", "{that}.options.request", "{that}.options.response"]
```

Comment:
This should also be updated, you're relying on an invoker rather than passing `response` now.